aws/cloud-formation/ecr/PROD-ECR.yaml

```diff
@@ -0,0 +1,49 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: ECS Fargate service in private subnets
+Parameters:
+  DigiroadDevArn:
+    Description: ARN of dev admin account
+    Type: String
+  CodebuildServiceRole:
+    Description: ARN of codebuild servicerole that pushes image.
```

Comment:
Description: ECS Fargate service in private subnets ei taida pitää paikkaansa enään